test/attributes/hx-disabled-elt.js

```diff
@@ -39,6 +39,18 @@ describe('hx-disabled-elt attribute', function() {
     fieldset.hasAttribute('disabled').should.equal(false)
   })
 
+  it('preserve pre-disabled elements', function() {
+    this.server.respondWith('GET', '/test', 'ok')
+    const b1 = make('<button hx-get="/test" hx-disabled-elt="#b2">Click Me!</button>')
+    const b2 = make('<button id="b2" disabled></button>')
+    b2.hasAttribute('disabled').should.equal(true)
+    b2.hasAttribute('data-disabled-by-htmx').should.equal(false)
+    b1.click()
+    b2.hasAttribute('disabled').should.equal(true)
```

Comment:
Suggestion: verify that `data-disabled-by-htmx` isn't set?  `b2.hasAttribute('data-disabled-by-htmx').should.equal(false)`. However, `data-disabled-by-htmx` isn't tested anywhere in this file.